scripts/jest/jestSequencer.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Sequencer = require('@jest/test-sequencer').default;
+
+class CustomSequencer extends Sequencer {
+  sort(tests) {
+    if (process.env.CIRCLE_NODE_TOTAL) {
```

Comment:
You'll want to clear this envar in the ReactClassEquivalence-test so that it can pick up the test files it runs, since it shells out to jest. We could also special case ReactClassEquivalence-test and run it on it's own.

Reply:
Ok I fixed it, forgot how to set an "empty" environment variable for a hot sec and got confused :D